integreat_cms/deepl_api/utils.py

```diff
@@ -139,3 +187,9 @@ def deepl_translation(self, request, content_objects, language_slug, form_class)
                         source_translation.title,
                     ),
                 )
+
+            # Update remaining DeepL usage for the region
+            request.region.deepl_usage_remaining = max(
+                0, request.region.deepl_usage_remaining - translation_word_count
+            )
+            request.region.save()
```

Comment:
Probably a smart region manager with enough scripting skills can trick this counter by performing all translations at the same time :smile: 
I'm not even sure whether we can use [`select_for_update()`](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#select-for-update) here since the region object is already queried from the database in the [`RegionMiddleware`](https://github.com/digitalfabrik/integreat-cms/blob/9ba3bfbe5a8a4185258e9d4ebb969974522b8897/integreat_cms/core/middleware/region_middleware.py#L57)... but maybe re-querying at this point in time still works?

Reply:
This appears to be working, with one caveat: importing `Region` at the top of the module crashes Django with `django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet`. Moving the import inside the class fixes the error, though obviously this is not ideal either.

Do you happen to know what the reason behind this is?